Dockerfile.mayhem

```diff
@@ -0,0 +1,15 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```

Comment:
My only complaint here is that we're installing afl from scratch when AFL docker images already exist.